src/user/user.service.spec.ts

```
@@ -1,18 +1,130 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { UserService } from './user.service';
+import { PrismaService } from '../prisma/prisma.service';
+import { UserDto } from './dto/user.dto';
+import { UserId, UserMail, UserName } from './types'; // Import branded types
+import { User as UserModelPrisma } from '@prisma/client'; // Prisma's User model type
```

ESLint

[error] 'UserModelPrisma' is defined but never used.

(@typescript-eslint/no-unused-vars)